charts/sk8s/values.yaml

```diff
@@ -5,25 +5,36 @@ eventDispatcher:
   replicaCount: 1
   image:
     repository: sk8s/event-dispatcher
-    tag: v0001
+    tag: v0002
```

Comment:
did you intend to change these to v0002 or were you testing that locally?

Reply:
I did, I pushed v0002 to DockerHub, we could move the whole build to v0002 at some point

Reply:
okay - I was thinking that the `docker.tag` property is still at `v0001`, but I guess that's not relevant in this case, because the helm install will always be pulling from DockerHub... or would it ever be pulling locally if using minikube and the images are already local?

Reply:
it does pull from local, so if we set the new version in the pom we also need to update the v0001 used in other places

Reply:
we might want to decouple the build of the invoker images (currently triggered by the overall maven build), since with the helm install, all that should be necessary is to have those invoker images also available (we will eventually start publishing those to DockerHub also), and then the samples can be run